### ISSUE TITLE
Install fails on rscala::scalaEval

### ISSUE THREAD

**Issue by ssaxe-usgs** [jeffreyhanson/geotrellis#1]
Install fails because functions in the zzz.R file are no longer exported from the 'rscala' package.  The first failure occurs at rscala::scaleEval in zzz.R, returning:

"Error : .onAttach failed in attachNamespace() for 'geotrellis', details:
  call: NULL
  error: 'scalaEval' is not an exported object from 'namespace:rscala'"

Looks like you haven't worked on this package for 1 year+ but hoping you have a remedy for this.  Would be very interested in testing this out.

**Comment by jeffreyhanson** [comment]
Hi, yeah that's the problem. I had a quick look at the docs for the latest version of _rscala_ and couldn't find a drop in replacement for `scalaEval`. This package was really just a proof-of-concept so I don't the time to properly update this package to work with the new version of _rscala_. If you really wanted to try it out, I'd recommend installing an archived version of _rscala_ and seeing if that works (https://cran.r-project.org/src/contrib/Archive/rscala/)?

**Comment by ssaxe-usgs** [comment]
OK, thanks, I might try that out.  The main thing I was interested in was finding a raster processing method that I can run through R that doesn't actually operate in-memory (want it in R so I can include it in a Shiny app).  Since your package leverages Scala, does it still operate in-memory since it's run through R?

**Comment by jeffreyhanson** [comment]
Yeah the functions in this package process the data in-memory (though the raster data are loaded into a Scala interpreter, and not the R interpreter so perhaps might be more memory efficient). Sorry I can't be of more help.

**Comment by ssaxe-usgs** [comment]
That's good to know.  Thanks.
